data/wanderers middle.txt

```diff
@@ -2027,14 +2027,18 @@ mission "Wanderers: Sestor: Quarg Help 2"
 	description "Return to <planet> with this small fleet of Quarg ships."
 	source "Alta Hai"
 	destination "Farpoint"
+	cargo "medical supplies" 2
+	blocked "You will need <capacity> to carry supplies back to Farpoint."
 	to offer
 		has "Wanderers: Sestor: Quarg Help 1: done"
 	on offer
 		conversation
 			`As you are coming in for a landing, you contact the Quarg and discover that Elias Hanover is indeed still living here. He meets you the moment you dock with the station, and you describe the situation. "This must be retaliation," he says. "They want to wipe Danforth and the Oathkeepers off the map to get revenge for him driving them out of their base on Avalon."`
-			`	He leaves to try to find help, and returns about fifteen minutes later. "I've got four Quarg ships who volunteered to help," he says. "That may not seem like a lot, but their ships pack quite a punch. Good luck, captain."`
-			`	You thank him for his assistance, and hurry to fire up your ship and return to human space. Hopefully Danforth's base has not already been overrun in your absence.`
+			`	He leaves to try to find help, and returns about fifteen minutes later. "I've got four Quarg ships who volunteered to help," he says. "That may not seem like a lot, but their ships pack quite a punch. I also have some relief supplies to help those who have been injured. Good luck, captain."`
+			`	You thank him for his assistance and the aid, and hurry to fire up your ship and return to human space. Hopefully Danforth's base has not already been overrun in your absence.`
```

Comment:
This addition reads a bit oddly to me.

Reply:
Suggestions welcomed.

`You thank him for his assistance, and for donating the aid. You hurry back to your ship to return to human space, hoping Danforth's base has not already been overrun in your absence.`?